src/common/Misc/WirePrimitives.cpp

```diff
@@ -17,10 +17,10 @@
 // OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
 // THE SOFTWARE.
 #include "WirePrimitives.h"
-#include <DirectXMath.h>
-using namespace DirectX;
+#include <glm/glm.hpp>
```

Comment:
Both **DirectXMath** and **GLM** are not needed here.
It is enough to delete `include` and `using namespace` here